=== instances.rb
==================================================================
--- instances.rb	(revision 795)
+++ instances.rb	(local)
@@ -59,7 +59,7 @@
         if in_params[:base64Encoded]
           userData = in_params[:userData]
         else
-          userData = CGI::escape(Base64.encode64(in_params[:userData]).strip())
+          userData = CGI::escape(Base64.encode64(in_params[:userData]).gsub(/\n/,"").strip())
         end
       else
         userData = nil
